compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -155,6 +155,10 @@ trait ResolverAstLoweringExt {
     fn legacy_const_generic_args(&self, expr: &Expr) -> Option<Vec<usize>>;
     fn get_partial_res(&self, id: NodeId) -> Option<PartialRes>;
     fn get_import_res(&self, id: NodeId) -> PerNS<Option<Res<NodeId>>>;
+    // Applies the resolution (if any) on 'source' and applies it
```

Comment:
nit: "applies ... and applies"

```suggestion
    // Clones the resolution (if any) on 'source' and applies it
```